juspay/hyperswitch
# refactor: retrieve extended card info config during business profile get call

## Type of Change



## Description

During business profile get call, send the extended card info config of merchant (public key and ttl) in the response so that dashboard can make use of it

### Additional Changes





## Motivation and Context



## How did you test it?

1. Create merchant account (get default profile_id)
2. Create api key
3. Use update endpoint of business profile to pass config.
```
curl --location 'http://localhost:8080/account/merchant_1716883344/business_profile/pro_5PdcjX4zYfC8X2PFjCel' \
--header 'Content-Type: application/json' \
--header 'Accept: application/json' \
--header 'api-key: test_admin' \
--data '{
    "extended_card_info_config": {
        "ttl_in_secs": 300,
        "public_key": "pub_key"
    }
}'
```
![image](https://github.com/juspay/hyperswitch/assets/70657455/67787f0e-724c-4bed-ae37-a2636093e481)

4. Retrieve the business profile and the config should be returned in the response

```
curl --location 'http://localhost:8080/account/merchant_1716883344/business_profile/pro_5PdcjX4zYfC8X2PFjCel' \
--header 'Content-Type: application/json' \
--header 'Accept: application/json' \
--header 'api-key: test_admin' \
--data ''
```

![image](https://github.com/juspay/hyperswitch/assets/70657455/eefcd267-ad39-447f-aa7a-8536537f935d)


## Checklist
